pkgs/development/compilers/elm/default.nix

```diff
@@ -39,6 +49,11 @@ let
                 hpack
               '';
               jailbreak = true;
+
+              description = "This is a small tool that reads .elmi files (Elm interface file) generated by the elm compiler. These files contain information about exposed definitions (e.g. custom types, type aliases and functions).";
```

Comment:
This description is a bit long and probably fits better into longDescription.

Reply:
All the descriptions are coppied from official source (either github repository or package manager information). I was not sure if it's OK to use custom one or not.